web/src/features/fireWeather/components/TimeOfInterestPicker.tsx

```diff
@@ -26,6 +26,7 @@ const TimeOfInterestPicker = (props: Props) => {
       label="Time of Interest (PST-08:00)"
       type="datetime-local"
       value={formatDateInPST(props.timeOfInterest, 'YYYY-MM-DDTHH:mm')}
+      helperText="Disclaimer: all data may not be available."
```

Comment:
I think what you mean is "not all data may be available" ?

Reply:
Okay will change that